### PR TITLE
add 'use strict'; to examples

### PR DESCRIPTION
**Pull request by vsemozhetbyt**
I am not sure what else non-module scripts should be addressed. If any, let me know to add them in another commit.

### CODE REVIEW

**Review by ebidel — APPROVED** [review]

**Comment by ebidel** [comment]
Should probably enforce this in https://github.com/GoogleChrome/puppeteer/blob/master/examples/.eslintrc.js

File: test/test.js [hunk 1]
```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+'use strict';
```
Comment (aslushnikov) [inline test/test.js]: why do we have it here? do we need to use 'use strict' in all of the source code?

**Comment by vsemozhetbyt** [comment]
@ebidel 
> Should probably enforce this...

Hopefully done in the second commit.

File: test/test.js (hunk 1 above)
Comment (vsemozhetbyt) [reply test/test.js]: I think this is the de-facto standard for non-module scripts if I get this right. Should the test check the API with the same semantics as users will use it in scripts?

Comment (ebidel) [reply test/test.js]: We only need it in non-imported code (tests, example). When users import `puppeteer` those are run as modules and have an implicit 'use strict'.

Comment (aslushnikov) [reply test/test.js]: > Should the test check the API with the same semantics as users will use it in scripts?

The `use strict` in the beginning of the test ensures that the `test.js` itself conforms to the strict mode; it doesn't imply anything about the puppeteer code. We also have a pretty tight eslint rules that should enforce us into good code practices, so `use strict` in the `test.js` doesn't give us anything.

Comment (vsemozhetbyt) [reply test/test.js]: Should I exclude this one from the commit?

Comment (aslushnikov) [reply test/test.js]: @vsemozhetbyt yeah, let's exclude

Comment (vsemozhetbyt) [reply test/test.js]: Done.

**Pull request merged**